apps/test/unit/storage/dataBrowser/dataVisualizer/VisualizerModalTest.js

```diff
@@ -198,4 +198,155 @@ describe('VisualizerModal', () => {
       ).to.deep.equal(['"xyz"', '"def"', '"123"', 'undefined', '123', 'true']);
     });
   });
+
+  describe('filterRecords', () => {
+    let wrapper;
+    beforeEach(() => {
+      wrapper = shallow(<VisualizerModal {...DEFAULT_PROPS} />);
+    });
+    it('works with numbers', () => {
+      let records = [
+        {id: 1, filterCol: 123, chartCol: 2},
+        {id: 2, filterCol: 456, chartCol: 3},
+        {id: 3, filterCol: 123, chartCol: 5},
+        {id: 4, filterCol: '456', chartCol: 7},
+        {id: 5, filterCol: 0, chartCol: 5}
+      ];
```

Comment:
since we're using this array everywhere, should we hoist this to the top of this `describe` block and merge all of the separate instances of `records` into a single array?

Reply:
It has different values for different tests, though. I think I prefer having a shorter list of records for each test rather than one big list to cover all tests. IMO that has the added benefit to readability of being able to see the test data and expected result in one place, rather than having to scroll back and forth